src/api/app/models/backend_package.rb

```diff
@@ -20,7 +20,7 @@ class BackendPackage < ApplicationRecord
   # this is called from the UpdatePackageMetaJob and clockwork
   def self.refresh_dirty
     Package.dirty_backend_package.pluck(:project_id).uniq.each do |project_id|
-      Project.find(project_id).delay(priority: 10).update_packages_if_dirty
+      Project::UpdatePackagesIfDirtyJob.perform_later(project_id)
```

Comment:
Why do you keep this *abstraction*? A job calls a class method of one model that calls a instance method of another class (that instantiates itself) to call an instance method on itself. What's wrong with calling `Project.find(project_id).update_packages_if_dirty.perform_later` directly?